neo/Prompt/Commands/Config.py

```diff
@@ -0,0 +1,205 @@
+                print("Smart contract emit Notify events on execution failure is now enabled")
+                return c1
+            elif c1 == 'off' or c1 == '0':
+                settings.set_emit_notify_events_on_sc_execution_error(False)
+                print("Smart contract emit Notify events on execution failure is now disabled")
+                return c1
+            else:
+                print("Cannot configure log. Please specify on|off")
+                return
+        else:
+            print("Cannot configure log. Please specify on|off")
+            return
+
+    def command_desc(self):
+        p1 = ParameterDesc('attribute', 'either "on"|"off" or 1|0')
+        return CommandDesc('sc-debug-notify', 'toggle Notify events on execution failure on or off', [p1])
+
+
+class CommandConfigVMLog(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        c1 = get_arg(arguments)
+        if c1 is not None:
+            c1 = c1.lower()
+            if c1 == 'on' or c1 == '1':
+                settings.set_log_vm_instruction(True)
+                print("VM instruction execution logging is now enabled")
+                return c1
+            elif c1 == 'off' or c1 == '0':
+                settings.set_log_vm_instruction(False)
+                print("VM instruction execution logging is now disabled")
+                return c1
+            else:
+                print("Cannot configure VM instruction logging. Please specify on|off")
+                return
+        else:
+            print("Cannot configure VM instruction logging. Please specify on|off")
+            return
+
+    def command_desc(self):
+        p1 = ParameterDesc('attribute', 'either "on"|"off" or 1|0')
+        return CommandDesc('vm-log', 'toggle VM instruction execution logging on or off', [p1])
+
+
+class CommandConfigNodeRequests(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        if len(arguments) in [1, 2]:
+            if len(arguments) == 2:
+                try:
+                    return NodeLeader.Instance().setBlockReqSizeAndMax(int(arguments[0]), int(arguments[1]))
+                except ValueError:
+                    print("Please specify a block request part and max size for each node, like 30 and 1000")
+                    return False
+            elif len(arguments) == 1:
+                return NodeLeader.Instance().setBlockReqSizeByName(arguments[0])
+        else:
+            print("Invalid number of arguments")
+            return False
+
+    def command_desc(self):
+        p1 = ParameterDesc('attribute', 'specify "slow"/"normal"/"fast" preset block request sizes, or specific block request size')
+        p2 = ParameterDesc('queue-size', 'specific max queue size. Required if custom block request size is specified', optional=True)
+        return CommandDesc('node-requests', 'configure the block request size', [p1, p2])
```

Comment:
It is good that I never used this setting before because that made it clear to me that the descriptions where not self explanatory. I think `configure block request settings` would make more sense as a description because only one of the parameters is `block request size` the other is queue size related.
```suggestion
         return CommandDesc('node-requests', 'configure block request settings', [p1, p2])
```

Reply:
I agree. This is a much better description. I couldn't think of any at the time. 👍